EventFilter/CTPPSRawToDigi/python/ctppsRawToDigi_cff.py

```diff
@@ -104,6 +104,7 @@
 totemDAQMappingESSourceXML_TotemTiming = cms.ESSource("TotemDAQMappingESSourceXML",
   verbosity = cms.untracked.uint32(10),
   subSystem = cms.untracked.string("TotemTiming"),
+  sampicSubDetId = cms.untracked.uint32(5),
```

Comment:
```suggestion
  sampicSubDetId = cms.uint32(5),
```

Suggestion from @davidlange6 actually

Reply:
This makes sense. If applied, it has to be propagated also elsewere in the PR.
And in any case, please fix it in the master first

Reply:
addressed in [deb4a01](https://github.com/cms-sw/cmssw/pull/38365/commits/deb4a010c508e4d98efedd1db44d0b917bf7c0c2)